app.js

```diff
@@ -2,22 +2,52 @@
 
 // List 3 expressions that surprised you or gave unexpected results, why?
 
+    // "2.5 * 'fish'" gave an unexpected result mainly because I had forgotten that NaN could come into play in a moment like this. 
+    // "'new york'[0]" was unexpected becuase I struggled to see 'new york' as an array, as opposed to a simple string.
```

Comment:
`"new york"` is a string and each character in a string has an index. Some of the things we can do with arrays we can also do with string!